STEERING-COMMITTEE.md

```diff
@@ -17,29 +17,36 @@ the way we run this committee, based on feedback from the community.
 - [Charter](#charter)
```

Comment:
Let's delete the first sentence about being "the ultimate authority", and link the second back to GOVERNANCE.md

Reply:
Done.